### DISCUSSION TITLE
Do we have approvals for adding or modifying part, build, PO?

### DISCUSSION THREAD

**Discussion by rishabhK96** [inventree/InvenTree#6119] [Q&A] [answered]
I am an administrator of the InvenTree instance running on my local server via Docker. We have two level of hierarchy in my organisation, one for adding, parts etc to the database and the other to validate the data entered is correct and approve it to go live. Earlier I saw this option but now can't find it OR it can be that I just hallucinated this feature.

Let me know if it is there or can be enabled via some plug-in.

**Comment by SchrodingersGat** [comment, marked as answer]
@rishabhK96 you may have dreamed that one :) There are some open discussions currently regarding approvals, but nothing has made it into codebase yet. 

The discussion also does not cover general data entry, allowing parts to "go live". It would be worth checking out where the approval plugin design is at, though.
